admin/client/App/screens/List/index.js

```diff
@@ -333,25 +333,25 @@ const ListView = React.createClass({
 			checkedItems: {},
 		});
 	},
-	deleteTableItem (item, e) {
-		if (e.altKey) {
+	deleteTableItem (item, event) {
```

Comment:
Taking `e` as the event argument is pretty consistent and standard, I'm OK with leaving this as is